UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/Visualizer/DrawOnTexture.cs

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking
```

Comment:
Please remove the `Demos.EyeTracking` from the namespace

Nit Pick/Style....Please start fields with a lowercase letter, not underscore.